### DISCUSSION TITLE
輪読会テーマ本

### DISCUSSION THREAD

**Discussion by ryutah** [dokurin/toy-box#37] [General]
次に読む本決めましょう

**Comment by ryutah** [comment]
[計算できるもの、計算できないもの ―実践的アプローチによる計算理論入門](https://www.amazon.co.jp/%E8%A8%88%E7%AE%97%E3%81%A7%E3%81%8D%E3%82%8B%E3%82%82%E3%81%AE%E3%80%81%E8%A8%88%E7%AE%97%E3%81%A7%E3%81%8D%E3%81%AA%E3%81%84%E3%82%82%E3%81%AE-%E2%80%95%E5%AE%9F%E8%B7%B5%E7%9A%84%E3%82%A2%E3%83%97%E3%83%AD%E3%83%BC%E3%83%81%E3%81%AB%E3%82%88%E3%82%8B%E8%A8%88%E7%AE%97%E7%90%86%E8%AB%96%E5%85%A5%E9%96%80-John-MacCormick/dp/4873119332/ref=sr_1_1?__mk_ja_JP=%E3%82%AB%E3%82%BF%E3%82%AB%E3%83%8A&crid=2RLB04764FDJ4&dchild=1&keywords=%E8%A8%88%E7%AE%97%E3%81%A7%E3%81%8D%E3%82%8B%E3%82%82%E3%81%AE+%E8%A8%88%E7%AE%97%E3%81%A7%E3%81%8D%E3%81%AA%E3%81%84%E3%82%82%E3%81%AE&qid=1633160122&s=books&sprefix=%E8%A8%88%E7%AE%97%E3%81%A7%E3%81%8D%E3%82%8B%E3%82%82%E3%81%AE%2Cstripbooks%2C251&sr=1-1) from @yoshioH

**Comment by ryutah** [comment]
[ITビジネスの契約実務](https://www.amazon.co.jp/dp/4785729031) from @yoshioH

**Comment by ryutah** [comment]
[n月刊ラムダノート Vol.1, No.1(2019)](https://www.lambdanote.com/collections/n/products/nmonthly-vol-1-no-1-2019) from @yoshioH

**Comment by ryutah** [comment]
[ゼロトラストネットワーク――境界防御の限界を超えるためのセキュアなシステム設計](https://www.oreilly.co.jp/books/9784873118888/) from @ryutah

**Comment by yoshioH** [comment]
[暗号と認証のしくみと理論がこれ1冊でしっかりわかる教科書](https://www.amazon.co.jp/dp/429712307X)
from @yoshioH

**Reply by ryutah** [reply]
> 9章 高機能な暗号技術
45 準同型暗号
46 秘密計算
47 ゼロ知識証明
48 量子コンピュータ

個人的に気になるところはこのあたり

**Comment by yoshioH** [comment]
[1日1問、半年以内に習得 シェル・ワンライナー160本ノック](https://www.amazon.co.jp/dp/4297122677) from @yoshioH

**Comment by ryutah** [comment]
[入門 監視――モダンなモニタリングのためのデザインパターン](https://www.oreilly.co.jp/books/9784873118642/) from @ryutah

**Comment by yoshioH** [comment]
[白と黒のとびら: オートマトンと形式言語をめぐる冒険](https://www.amazon.co.jp/dp/4130633570) from @yoshioH

**Comment by ryutah** [comment]
[TCP技術入門 ――進化を続ける基本プロトコル (WEB+DB PRESS plusシリーズ)](https://www.amazon.co.jp/dp/429710623X/) from @yoshioH

**Comment by yoshioH** [comment]
[ビヨンド ソフトウェア アーキテクチャ Object Oriented Selection Classics](https://www.amazon.co.jp/%E3%82%BD%E3%83%95%E3%83%88%E3%82%A6%E3%82%A7%E3%82%A2-%E3%82%A2%E3%83%BC%E3%82%AD%E3%83%86%E3%82%AF%E3%83%81%E3%83%A3-Oriented-Selection-Classics-ebook/dp/B015WE7HQ8) from @ryutah

**Comment by ryutah** [comment]
[モノリスからマイクロサービスへ――モノリスを進化させる実践移行ガイド](https://www.oreilly.co.jp/books/9784873119311/) from @ryutah

**Comment by yoshioH** [comment]
[マイクロサービスパターン［実践的システムデザインのためのコード解説］](https://www.amazon.co.jp/%E3%83%9E%E3%82%A4%E3%82%AF%E3%83%AD%E3%82%B5%E3%83%BC%E3%83%93%E3%82%B9%E3%83%91%E3%82%BF%E3%83%BC%E3%83%B3%EF%BC%BB%E5%AE%9F%E8%B7%B5%E7%9A%84%E3%82%B7%E3%82%B9%E3%83%86%E3%83%A0%E3%83%87%E3%82%B6%E3%82%A4%E3%83%B3%E3%81%AE%E3%81%9F%E3%82%81%E3%81%AE%E3%82%B3%E3%83%BC%E3%83%89%E8%A7%A3%E8%AA%AC%EF%BC%BD-impress-gear%E3%82%B7%E3%83%AA%E3%83%BC%E3%82%BA-Chris-Richardson-ebook/dp/B086JJNDKS/ref=sr_1_1?__mk_ja_JP=%E3%82%AB%E3%82%BF%E3%82%AB%E3%83%8A&crid=2BFFWPCZN99H6&dchild=1&keywords=%E3%83%9E%E3%82%A4%E3%82%AF%E3%83%AD%E3%82%B5%E3%83%BC%E3%83%93%E3%82%B9%E3%83%91%E3%82%BF%E3%83%BC%E3%83%B3&qid=1633159456&s=digital-text&sprefix=%E3%83%9E%E3%82%A4%E3%82%AF%E3%83%AD%2Cdigital-text%2C347&sr=1-1) from @ryutah

**Comment by ryutah** [comment]
[エンジニアの知的生産術 ―効率的に学び、整理し、アウトプットする](https://www.amazon.co.jp/%E3%82%A8%E3%83%B3%E3%82%B8%E3%83%8B%E3%82%A2%E3%81%AE%E7%9F%A5%E7%9A%84%E7%94%9F%E7%94%A3%E8%A1%93-%E2%80%95%E5%8A%B9%E7%8E%87%E7%9A%84%E3%81%AB%E5%AD%A6%E3%81%B3%E3%80%81%E6%95%B4%E7%90%86%E3%81%97%E3%80%81%E3%82%A2%E3%82%A6%E3%83%88%E3%83%97%E3%83%83%E3%83%88%E3%81%99%E3%82%8B-WEB-PRESS-plus-ebook/dp/B07JFRF6MW/ref=sr_1_1?__mk_ja_JP=%E3%82%AB%E3%82%BF%E3%82%AB%E3%83%8A&crid=1228420K0JF9D&dchild=1&keywords=%E3%82%A8%E3%83%B3%E3%82%B8%E3%83%8B%E3%82%A2%E3%81%AE%E7%9F%A5%E7%9A%84%E7%94%9F%E7%94%A3%E8%A1%93&qid=1633159797&sprefix=%E3%82%A8%E3%83%B3%E3%82%B8%E3%83%8B%E3%82%A2%E3%81%AE%E7%9F%A5%E7%9A%84%2Caps%2C683&sr=8-1) from @ryutah

**Reply by ryutah** [reply]
* 第一章
  * 大学に入りなおすべき？
  * Matzのソースコードの読み方
* 第二章
  * やる気が出ない人の６５％はタスクを１つに絞れていない
  * タイムボックス
* 第三章
  * 記憶と筋肉の共通点
  * 知識を構造化する20のルール
  * 難易度の自動調節
* 第四章
  * あなたの普段の読む速度は？
  * １ページ２秒以下の「見つける」読み方
* 第五章
  * 多すぎる情報をどうまとめるか
  * 「考えがまとまらない」と「部屋が片付かない」は似ている
* 第六章
  * 言語化を促す方法
  * 再び耕す
* 第七章
  * 意思決定の正しさ
  * 自分経営戦略

**Reply by ryutah** [reply]
## 順番

1. @ryutah 
2. @yoshioH 
3. @second10 
4. @itor319 

## 読書ページ

* Total（1 ~ 7章）
  * 240P
* Per day
  * 30P
* Finish week
  * 8 week

## スケジュール

* 一週目
  * ~ 情報収集の３つの方法
* 二週目
  * 抽象とはなにか ~
* 三週目
  * １つのタスクのやる気を出す ~ 
* 四週目
  * 知識を長持ちさせる感覚反復法 ~

**Reply by ryutah** [reply]
~ までより、~ から読み始めるってしたほうが次週どこから始まるかとかわかりやすくていい気がした

**Comment by yoshioH** [comment]
[Engineers in VOYAGE ― 事業をエンジニアリングする技術者たち](https://www.amazon.co.jp/Engineers-VOYAGE-%E4%BA%8B%E6%A5%AD%E3%82%92%E3%82%A8%E3%83%B3%E3%82%B8%E3%83%8B%E3%82%A2%E3%83%AA%E3%83%B3%E3%82%B0%E3%81%99%E3%82%8B%E6%8A%80%E8%A1%93%E8%80%85%E3%81%9F%E3%81%A1-%E6%A0%AA%E5%BC%8F%E4%BC%9A%E7%A4%BEVOYAGE-GROUP-ebook/dp/B08GSQ4BL3) from @ryutah

**Comment by ryutah** [comment]
[The DevOps ハンドブック 理論・原則・実践のすべて](https://www.amazon.co.jp/DevOps-%E3%83%8F%E3%83%B3%E3%83%89%E3%83%96%E3%83%83%E3%82%AF-%E7%90%86%E8%AB%96%E3%83%BB%E5%8E%9F%E5%89%87%E3%83%BB%E5%AE%9F%E8%B7%B5%E3%81%AE%E3%81%99%E3%81%B9%E3%81%A6-%E3%82%B8%E3%83%BC%E3%83%B3%E3%83%BB%E3%82%AD%E3%83%A0-ebook/dp/B073PRMWR5/ref=sr_1_3?__mk_ja_JP=%E3%82%AB%E3%82%BF%E3%82%AB%E3%83%8A&dchild=1&keywords=The+Dev+Ops&qid=1633159748&sr=8-3) from @ryutah

**Comment by yoshioH** [comment]
[エンジニアリング組織論への招待　～不確実性に向き合う思考と組織のリファクタリング ](https://www.amazon.co.jp/%E3%82%A8%E3%83%B3%E3%82%B8%E3%83%8B%E3%82%A2%E3%83%AA%E3%83%B3%E3%82%B0%E7%B5%84%E7%B9%94%E8%AB%96%E3%81%B8%E3%81%AE%E6%8B%9B%E5%BE%85-%EF%BD%9E%E4%B8%8D%E7%A2%BA%E5%AE%9F%E6%80%A7%E3%81%AB%E5%90%91%E3%81%8D%E5%90%88%E3%81%86%E6%80%9D%E8%80%83%E3%81%A8%E7%B5%84%E7%B9%94%E3%81%AE%E3%83%AA%E3%83%95%E3%82%A1%E3%82%AF%E3%82%BF%E3%83%AA%E3%83%B3%E3%82%B0-%E5%BA%83%E6%9C%A8-%E5%A4%A7%E5%9C%B0-ebook/dp/B079TLW41L/ref=sr_1_1?__mk_ja_JP=%E3%82%AB%E3%82%BF%E3%82%AB%E3%83%8A&crid=3C4MPMVZ4OWE7&dchild=1&keywords=%E3%82%A8%E3%83%B3%E3%82%B8%E3%83%8B%E3%82%A2%E7%B5%84%E7%B9%94%E8%AB%96&qid=1633159563&s=digital-text&sprefix=%E3%82%A8%E3%83%B3%E3%82%B8%E3%83%8B%E3%82%A2%E7%B5%84%E7%B9%94%2Cdigital-text%2C287&sr=1-1) from @ryutah

**Reply by ryutah** [reply]
これをやっていきましょうかー

**Comment by ryutah** [comment]
[SCRUMMASTER THE BOOK 優れたスクラムマスターになるための極意――メタスキル、学習、心理、リーダーシップ](https://www.amazon.co.jp/SCRUMMASTER-BOOK-%E5%84%AA%E3%82%8C%E3%81%9F%E3%82%B9%E3%82%AF%E3%83%A9%E3%83%A0%E3%83%9E%E3%82%B9%E3%82%BF%E3%83%BC%E3%81%AB%E3%81%AA%E3%82%8B%E3%81%9F%E3%82%81%E3%81%AE%E6%A5%B5%E6%84%8F%E2%80%95%E2%80%95%E3%83%A1%E3%82%BF%E3%82%B9%E3%82%AD%E3%83%AB%E3%80%81%E5%AD%A6%E7%BF%92%E3%80%81%E5%BF%83%E7%90%86%E3%80%81%E3%83%AA%E3%83%BC%E3%83%80%E3%83%BC%E3%82%B7%E3%83%83%E3%83%97-Zuzana-Sochova/dp/4798166855/ref=sr_1_1?__mk_ja_JP=%E3%82%AB%E3%82%BF%E3%82%AB%E3%83%8A&crid=23UKVPQBSBRDB&dchild=1&keywords=scrum+master&qid=1633159691&sprefix=Scrum+master%2Caps%2C277&sr=8-1) from @ryutah

**Comment by yoshioH** [comment]
[テストから見えてくるグーグルのソフトウェア開発](https://www.amazon.co.jp/%E3%83%86%E3%82%B9%E3%83%88%E3%81%8B%E3%82%89%E8%A6%8B%E3%81%88%E3%81%A6%E3%81%8F%E3%82%8B%E3%82%B0%E3%83%BC%E3%82%B0%E3%83%AB%E3%81%AE%E3%82%BD%E3%83%95%E3%83%88%E3%82%A6%E3%82%A7%E3%82%A2%E9%96%8B%E7%99%BA-%E3%82%B8%E3%82%A7%E3%83%BC%E3%83%A0%E3%82%BA-%E3%82%A6%E3%82%A3%E3%83%86%E3%82%AB%E3%83%BC%EF%BC%9B%E3%82%B8%E3%82%A7%E3%83%BC%E3%82%BD%E3%83%B3-%E3%82%A2%E3%83%BC%E3%83%9C%E3%83%B3%EF%BC%9B%E3%82%B8%E3%82%A7%E3%83%95-%E3%82%AD%E3%83%A3%E3%83%AD%E3%83%BC%E3%83%AD-ebook/dp/B00IE3B522/ref=sr_1_1?__mk_ja_JP=%E3%82%AB%E3%82%BF%E3%82%AB%E3%83%8A&crid=306VS1GXJSZF7&dchild=1&keywords=%E3%82%B0%E3%83%BC%E3%82%B0%E3%83%AB%E3%81%AE%E3%82%BD%E3%83%95%E3%83%88%E3%82%A6%E3%82%A7%E3%82%A2%E9%96%8B%E7%99%BA&qid=1633159643&sprefix=%E3%82%B0%E3%83%BC%E3%82%B0%E3%83%AB%E3%81%AE%E3%82%BD%E3%83%95%E3%83%88%E3%82%A6%E3%82%A7%E3%82%A2%2Cdigital-text%2C292&sr=8-1) from @ryutah

**Comment by ryutah** [comment]
[セキュア・バイ・デザイン](https://www.amazon.co.jp/%E3%82%BB%E3%82%AD%E3%83%A5%E3%82%A2%E3%83%BB%E3%83%90%E3%82%A4%E3%83%BB%E3%83%87%E3%82%B6%E3%82%A4%E3%83%B3-Dan-Bergh-Johnsson/dp/483997599X)

![image](https://user-images.githubusercontent.com/6662577/147383244-c89fa0f9-2e04-4160-9543-06cc2dde4afa.png)

**Comment by yoshioH** [comment]
[カーハッカーズ・ハンドブック―車載システムの仕組み・分析・セキュリティ](https://www.oreilly.co.jp/books/9784873118239/)

**Comment by ryutah** [comment]
[実践ソフトウェアエンジニアリング (第9版) ](https://www.amazon.co.jp/dp/B09MSZQB5Y/?coliid=I1M6EVQPWUCGV5&colid=1D9NKLS8D7QOI&psc=0&ref_=lv_ov_lig_dp_it)

---

なんかすごい重版されてるのでまあいい本なんじゃないかと思われ

**Comment by ryutah** [comment]
[Software Engineering at Google: Lessons Learned from Programming Over Time (English Edition) 1st 版,](https://www.amazon.co.jp/dp/B0859PF5HB/?coliid=I3FMPMBPSUU23X&colid=1D9NKLS8D7QOI&psc=0&ref_=lv_ov_lig_dp_it)

**Comment by ryutah** [comment]
[リファクタリング 既存のコードを安全に改善する（第2版）](https://www.amazon.co.jp/%E3%83%AA%E3%83%95%E3%82%A1%E3%82%AF%E3%82%BF%E3%83%AA%E3%83%B3%E3%82%B0-%E6%97%A2%E5%AD%98%E3%81%AE%E3%82%B3%E3%83%BC%E3%83%89%E3%82%92%E5%AE%89%E5%85%A8%E3%81%AB%E6%94%B9%E5%96%84%E3%81%99%E3%82%8B%EF%BC%88%E7%AC%AC2%E7%89%88%EF%BC%89-%EF%BC%AD%EF%BD%81%EF%BD%92%EF%BD%94%EF%BD%89%EF%BD%8E%EF%BC%A6%EF%BD%8F%EF%BD%97%EF%BD%8C%EF%BD%85%EF%BD%92-ebook/dp/B0831M1RK5/ref=sr_1_2?__mk_ja_JP=%E3%82%AB%E3%82%BF%E3%82%AB%E3%83%8A&crid=JSKG05VCLVFT&keywords=%E3%83%AA%E3%83%95%E3%82%A1%E3%82%AF%E3%82%BF%E3%83%AA%E3%83%B3%E3%82%B0&qid=1648887148&s=digital-text&sprefix=%E3%83%AA%E3%83%95%E3%82%A1%E3%82%AF%E3%82%BF%E3%83%AA%E3%83%B3%E3%82%B0%2Cdigital-text%2C261&sr=1-2)

**Comment by yoshioH** [comment]
[地球の歩き方 ムー-異世界(パラレルワールド)の歩き方ー超古代文明 オーパーツ 聖地 UFO UMA](https://www.amazon.co.jp/%E5%9C%B0%E7%90%83%E3%81%AE%E6%AD%A9%E3%81%8D%E6%96%B9-%E3%83%A0%E3%83%BC-%E7%95%B0%E4%B8%96%E7%95%8C-%E3%83%91%E3%83%A9%E3%83%AC%E3%83%AB%E3%83%AF%E3%83%BC%E3%83%AB%E3%83%89-%E3%81%AE%E6%AD%A9%E3%81%8D%E6%96%B9%E3%83%BC%E8%B6%85%E5%8F%A4%E4%BB%A3%E6%96%87%E6%98%8E-%E3%82%AA%E3%83%BC%E3%83%91%E3%83%BC%E3%83%84/dp/4058017163/ref=pd_sbs_sccl_2/355-5699290-1970907?pd_rd_w=M7xfF&pf_rd_p=eb649620-718d-4f3b-9aaa-29b215914196&pf_rd_r=6GXBF4VHZZB2C0HBZ67X&pd_rd_r=f6e429e2-fe80-48bf-9fe4-2a94eacc17f3&pd_rd_wg=xWEal&pd_rd_i=4058017163&psc=1)

**Comment by yoshioH** [comment]
[マスターショット100 低予算映画を大作に変える撮影術](https://www.amazon.co.jp/dp/4845911655)
[映画表現の教科書 ─名シーンに学ぶ決定的テクニック100](https://www.amazon.co.jp/dp/4845912929)
